code/modules/power/singularity/emitter.dm

```diff
@@ -1,6 +1,6 @@
 /obj/machinery/power/emitter
-	name = "Emitter"
-	desc = "A heavy duty industrial laser.\n<span class='notice'>Alt-click to rotate it clockwise.</span>"
+	name = "emitter"
+	desc = "A heavy-duty industrial laser used in containment fields and power generation.\n<span class='notice'>Alt-click to rotate it clockwise.</span>"

```

Comment:
Do we have to add that on? I always assumed emitters and industrial laser technology could be used for other things in the universe?